etc/openqa/openqa.ini

```diff
@@ -108,7 +108,7 @@
 ## if 'scm' is set to 'git' in [global]
 #[scm git]
 ## whether to automatically commit needles created in the editor back to git
-#git_auto_commit = yes
+#git_auto_commit = no
```

Comment:
if git features are enabled by default why is that changed?

Reply:
what we're trying to achieve (AIUI) is for `git_auto_push` and `git_auto_update` to be default-enabled, but `git_auto_commit` to be default-disabled, because that's how things were historically and also what we think is appropriate (because we perceive `git_auto_commit` to be a more intrusive thing than the other two).

the tricky part is implementing this in such a way that existing config files maintain their meaning (i.e. `scm = git` turns `git_auto_commit` on, but without `scm = git`, it's off).

again, AIUI.

Reply:
> what we're trying to achieve (AIUI) is for `git_auto_push` and `git_auto_update` to be default-enabled, but `git_auto_commit` to be default-disabled, because that's how things were historically and also what we think is appropriate (because we perceive `git_auto_commit` to be a more intrusive thing than the other two).

I think you mean `git_auto_clone` instead of `git_auto_push` which does not exist, only `do_push`. But yes to the other points

> the tricky part is implementing this in such a way that existing config files maintain their meaning (i.e. `scm = git` turns `git_auto_commit` on _even without an explicit `git_auto_commit` setting_, but without `scm = git`, it's off).
> 
> again, AIUI.

Yes, correct

Reply:
edited, thanks.

Reply:
got it